pinot-query-planner/src/main/java/org/apache/pinot/calcite/rel/hint/PinotHintOptions.java

```diff
@@ -62,6 +62,7 @@ public static class AggregateOptions {
     public static final String GROUP_TRIM_SIZE = "group_trim_size";
 
     public static final String MAX_INITIAL_RESULT_HOLDER_CAPACITY = "max_initial_result_holder_capacity";
+    public static final String MSE_MAX_INITIAL_RESULT_HOLDER_CAPACITY = "mse_max_initial_result_holder_capacity";
```

Comment:
@Jackie-Jiang : do you have any recommendation for the hint name?

Reply:
@shauryachats : this hint is not required, since the current hint only controls this value for MSE anyways. (i.e. both of them control the same thing)